contracts/governance/StkTruToken.sol

```diff
@@ -261,6 +281,16 @@ contract StkTruToken is VoteToken, ClaimableContract, ReentrancyGuard {
         _claim(tfusd);
     }
 
+    /**
+     * @dev Claim rewards for specific token
+     * Allows account to claim specific token to save gas
+     * @param token Token to claim rewards for
+     */
+    function claimRewards(IERC20 token) external distribute updateRewards(msg.sender, token) {
```

Comment:
Remove distribute modifier & restrict to only reward tokens.
Add claimTRU() function which calls distribute.

Reply:
add tests for this